app/views/job_invocations/_form.html.erb

```diff
@@ -81,5 +84,20 @@
     </fieldset>
   </div>
 
+  <!-- modal window -->
```

Comment:
maybe extract this to partial or maybe even better, extract it to helper that would receive arguments like path, title, content, so it can be reused somewhere else?